### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ int8_t update();
 ![DemoFrame1](https://github.com/squix78/esp8266-oled-ssd1306/raw/master/resources/DemoFrame1.jpg)
 
 This frame shows three things:
- * How to draw an xbm image
+ * How to draw a xbm image
```
Comment (marcelstoer) [inline README.md:404]:
```suggestion
 * How to draw an XMB image
```

**Comment by marcelstoer** [comment]
Thanks

**Comment by thijstriemstra** [comment]
@marcelstoer can you put out a new release, pretty please?

**Comment by marcelstoer** [comment]
Yep, had actually wanted to do this for a while. Release 4.2.1 is out and should soon be picked up by the Arduino & PlatformIO library indexers.

**Pull request merged**